confChamber.py

```diff
@@ -107,7 +107,7 @@
     try:
         vfatTree = r.TTree('vfatTree','Tree holding VFAT Configuration Parameters')
         vfatTree.ReadFile(options.vfatConfig)
-        dict_readBack = { "vt1":"VThreshold1", "trimRange":"ContReg3" }
+        dict_readBack = { "vt1":"VThreshold1", "trimRange":"ContReg3", "vfatID":"ChipID1" }
```

Comment:
I guess this address isn't used to read then (also l71, l91)?
This was where I was unclear whether you were getting just the 8 bits corresponding to `ChipID1` or the full `chipID`, as returned by `getChipID`

Reply:
if it would be more clear I can change the trip here from `ChipID1` to just `ChipID`

Reply:
Probably better

Reply:
Done